avalon/tools/loader/app.py

```diff
@@ -418,12 +425,11 @@ def show(debug=False, parent=None, use_context=False):
         lib.refresh_group_config_cache()
 
         window = Window(parent)
-        window.show()
         window.setStyleSheet(style.load_stylesheet())
+        window.show()
```

Comment:
Is there a reason for these lines to be swapped? Double check that. Because some of these lines have swapped before I believe to fix stylesheet issues in specific hosts. Make sure they are the correct order, maybe check "blame/history" to see what's intended.

Reply:
There is reason. PySide / PySide2 / PyQt4 / PyQt5, each has different approach to propagation of stylesheets. One have issues when stylesheets are applied after show and other have issues if they are applied before show. Issue is not with widget itself but to it's children widgets (If they don't have set parent). With setting stylesheets before show and adding `parent` to initialization of widgets was only working way cross all Python's Qt frameworks.

- You're right that each widget should have set parent to make it 100% working.
- @davidlatwe it was your commit which changed the order of these lines. Can you check this works now in Houdini? To be at least 99% sure this works.